packages/studio/frontend/src/views/FlowEditor/inspector/index.tsx

```diff
@@ -0,0 +1 @@
+export {}
```

Comment:
Is this file necessary?

Reply:
it's a placeholder file, things will be added to it on the next ticket im working on so i thought I would leave it here for reference.